### DISCUSSION TITLE
[Fullstack] redwoodjs + typescript + bootstrap 4

### DISCUSSION THREAD

**Discussion by TristanWYL** [realworld-apps/realworld#937] [WIP Implementations]
Currently WIP.  All contributions are welcome and appreciated!

Visit the repo [here](https://github.com/TristanWYL/redwood-realworld-example-app).

The technical stacks:
 - [redwoodjs](https://github.com/redwoodjs/redwood)
 - [bootstrap 4](https://github.com/twbs/bootstrap/tree/v4-dev)
 - typescript

**Comment by geromegrignon** [comment]
Awesome, thanks for your contribution! 🚀
